_config/Payment.yaml

```diff
@@ -3,4 +3,4 @@ name: Payment
 
 Director:
   rules:
-    'payment/$MethodName/$Action/$ID': 'PaymentProcessor_GatewayHosted'
+    'PaymentProcessor_GatewayHosted//$Action!/$ID/$OtherID': 'PaymentProcessor_GatewayHosted'
```

Comment:
I don't think you need to explicitly set the shift points and so on.

Does this work with just:
`'PaymentProcessor_GatewayHosted': 'PaymentProcessor_GatewayHosted'`?